lib/utils/oauth2_util.ex

```diff
@@ -32,4 +33,9 @@ defmodule EaRestaurantDataLoader.Oauth2Util do
         {:error, message_description}
     end
   end
+
+  def get_encode_struct(encode_type, password, encoded_password) do
+    structs = %{"base64" => %Base64{password: password, encoded_password: encoded_password}}
```

Comment:
Move this map to `lib/utils/aplication_util.ex`. Prefer atoms over strings for in elixir whenever possible.

`get_encode_struct` should be renamed to something else that makes more sense, maybe `build_password_encoder_struct`.

`get_encode_struct` should within `lib/utils/aplication_util.ex` since it doesn't need anything from the `lib/utils/oauth2_util.ex`.

`structs` should be a map outside of `get_encode_struct`. in order to set properties to a struct use `Map.put/3`

Reply:
Done.